clients/cobol-lsp-vscode-extension/CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.14.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.13.0...0.14.0) (2020-09-10)
+
+#### Added
+
+- Outline view support
+- Breadcrumb support
+- Provide default COBOL code snippets
+- Underscore the copy statement if it contains a syntax error
+
+#### Fixed
+- Allow multi-line errors underscoring GH-486
+- Fix a typo in DIVIDE-REMAINDER code snippets
+- Fix checking program ID with quotes
+- Fix incorrect indicator area error message
+- Fix syntax analysis termination with unexpected copy statements GH-464
```

Comment:
In this context, the list elements should begin with a noun

Reply:
- Outline view support
- Breadcrumb support
- Default COBOL code snippets
- Underscoring the copy statement if it contains a syntax error

#### Fixed
- Multi-line errors underscoring GH-486
- A typo in DIVIDE-REMAINDER code snippets
- Program ID check with quotes
- Incorrect indicator area error message
- Syntax analysis termination with unexpected copy statements GH-464

Reply:
omg - nice exercise :-D